File: app/src/main/java/com/raiffeisen/backend/app/repository/model/Transaction.java
package com.raiffeisen.backend.app.repository.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Transaction {

    private Long accountFrom;
    private Long accountTo;
    private BigDecimal volume;
    private String fromName;
    private Long date;
    private boolean iscash;

}

File: Android/app/src/main/java/com/github/ivanshafran/raif/data/model/Transaction.kt
package com.github.ivanshafran.raif.data.model

import com.google.gson.annotations.SerializedName
import java.math.BigDecimal

class Transaction(
    @SerializedName("accountFrom")
    val accountFrom: Long,
    @SerializedName("fromName")
    val fromName: String,
    @SerializedName("accountTo")
    val accountTo: Long,
    @SerializedName("volume")
    val volume: BigDecimal
)
